fandogh_cli/utils.py

```diff
@@ -26,3 +31,23 @@ def do_every(seconds, task, while_condition=lambda: True):
     while while_condition():
         task()
         sleep(seconds)
+
+
+def makedirs(name, mode=0777, exist_ok=True):
```

Comment:
it's too open

Reply:
I copied from python2 library, what permission do you think is suitable?

Reply:
probably 0660